packages/fake-driver/lib/commands/alert.js

```diff
@@ -1,82 +1,73 @@
 import {errors} from 'appium/driver';
 
-/**
- * @template {Class<import('../types').IContextsCommands & import('@appium/types').IFindCommands>} T
- * @param {T} Base
- * @returns {Class<AlertCommands>}
- */
-export function AlertMixin(Base) {
+export default {
```

Comment:
why `export default` instead of `export async function` for each one?

Reply:
six and 1/2 dozen I guess

Reply:
@jlipps I could go either way here. given these aren't intended to be imported by anything other than `commands/index.js`, if you have a preference I can change it.

Reply:
I'll change it later if you wish